cql-engine-fhir/src/main/java/org/opencds/cqf/cql/model/Dstu2FhirModelResolver.java

```diff
@@ -6,7 +6,8 @@
 import java.lang.reflect.Field;
 import java.util.Calendar;
 
-import org.hl7.fhir.instance.model.*;
+import org.hl7.fhir.dstu2.model.*;
+import org.hl7.fhir.instance.model.AnnotatedUuidType;
```

Comment:
The bottom class here is ours. We should move it to org.hl7.fhir.dstu2.model.AnnotatedUuidType to preserve symmetry.